src/status_im/transport/message/v1/contact.cljs

```diff
@@ -76,8 +76,8 @@
     (let [message-id (transport.utils/message-id this)
           public-keys (remove nil? (map :public-key (vals (:contacts/contacts db))))]
       (handlers-macro/merge-fx cofx
-                               (protocol/multi-send-with-pubkey {:public-keys public-keys
-                                                                 :payload     this}))))
+                               (protocol/multi-send-by-pubkey {:public-keys public-keys
```

Comment:
Looks like these changes have been merged already or not related to this PR?

Reply:
yes I made an odd commit